react/utils/pixel.ts

```diff
@@ -10,15 +13,13 @@ export enum EventType {
 }
 
 export function handleProductClick(push: (data: any) => void, page: string) {
-  return (productId: string, position: number) =>
+  return (products: ISearchProduct[], id: string, position: number) =>
     push({
+      id,
       page,
       event: EVENT_NAME,
       eventType: EventType.ProductClick,
-      product: {
-        productId,
-        position,
-      },
```

Comment:
we cannot remove information that currently already exists in the events, as you did here removing the clicked product, as this would break our analytics and also our definition of product relevance based on clicks